RELEASE-NOTES.md

```diff
@@ -9,6 +9,7 @@
   please update your code:
   for instance, change `->addNewReference( [ $x, $y ] )` to `->addNewReference( $x, $y )`,
   and `->addNewReference( $snaks )` to `->addNewReference( ...$snaks )`.
+* `Statement`, `Reference`, `SnakList` and `Snak` no longer depend on the `Hashable` and `Immutable` interfaces from `DataValues/DataValues`.
```

Comment:
I find “no longer depend on” a strangle way to phrase this… they no longer implement those interfaces, and that’s a potentially breaking change to users of these classes. The current phrasing makes it sound too harmless, in my opinion – it sounds like the same functionality is being provided, just without an internal dependency.

Reply:
I was under the impression we were providing the same functionality in each of the implementing classes already. But of course you're right in case people decide to check for `Hashable` or `Implementable` in their code then it is a breaking change,  It just didn't cross my mind at the time.